vividus-plugin-rest-api/src/main/java/org/vividus/steps/api/HttpRequestSteps.java

```diff
@@ -77,6 +79,32 @@ public void request(DataWrapper content)
         httpTestContext.putRequestEntity(requestEntity);
     }
 
+    /**
+     * Sets application/x-www-form-urlencoded request entity that will be used while executing request.
+     * HTTP request header with name 'Content-Type' and value 'application/x-www-form-urlencoded; charset=UTF-8'
+     * is set.
+     * <div>Example:</div>
+     * <code>
+     *   <br>Given form data request:
+     *   <br>|name      |value |
+     *   <br>|firstName |Ivan  |
+     *   <br>|lastName  |Ivanov|
+     * </code>
+     * <br>
+     * <br>where
+     * @param parameters ExamplesTable representing list of parameters with columns "name" and "value" specifying
+     *                   form data request
+     */
+    @Given("form data request: $parameters")
+    public void putUrlEncodedRequest(ExamplesTable parameters)
+    {
+        List<BasicNameValuePair> listOfParams = parameters.getRowsAsParameters(true).stream()
+                .map(row -> new BasicNameValuePair(row.valueAs(NAME, String.class), row.valueAs(VALUE, String.class)))
+                .collect(toList());
+        httpTestContext.putRequestEntity(
+                new UrlEncodedFormEntity(listOfParams, StandardCharsets.UTF_8));
```

Comment:
it looks like it can be inlined, also it may look nicer if we use 
...
.collect(colllectingAndThen(toList(), params -> new UrlEncodedFormEntity(params, StandardCharsets.UTF_8)));
httpTestContext.putRequestEntity(requestEntity)